task/task_service_test.go

```diff
@@ -107,46 +107,37 @@ func Test_Task_TastService_Create_Wait(t *testing.T) {
 	newTaskService := NewTaskService(DefaultTaskServiceConfig())
 
 	action := func() error {
-		fmt.Printf("%#v\n", 2)
-
+		// Just something to do, so the task blocks
 		time.Sleep(1 * time.Second)
-		fmt.Printf("%#v\n", 7)
 
 		return nil
 	}
 
-	fmt.Printf("%#v\n", 1)
```

Comment:
some renaming I did while playing around here. Not really needed, but might make it more readable